css/style.css

```diff
@@ -0,0 +1,269 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Chango&family=Poppins:wght@300&display=swap');
+
+@import url('https://fonts.googleapis.com/css2?family=Chango&family=Poppins:wght@300&display=swap');
+ 
+*{
+	margin:0px;
+	
+	box-sizing: border-box;
+	font-family: 'Poppins', sans-serif;
+	
+	
```

Comment:
Leaving spaces between each styling would throw a linter error, it's best to avoid gaps between each styling to keep your code neat and tidy👍